claim/testdata/claim.allfields.json

```diff
@@ -1 +1 @@
-{"schemaVersion":"v1.0.0-WD","installation":"my_claim","revision":"revision","created":"1983-04-18T01:02:03.000000004Z","modified":"1983-04-18T01:02:03.000000004Z","bundle":{"schemaVersion":"schemaVersion","name":"mybun","version":"v0.1.0","description":"this is my bundle","invocationImages":null},"result":{"message":"result message","action":"install","status":"underway"},"parameters":{"myparam":"myparamvalue"},"outputs":{"myoutput":"myoutputvalue"},"custom":["anything goes"]}
\ No newline at end of file
+{"schemaVersion":"v1.0.0-WD","installation":"my_claim","revision":"revision","created":"1983-04-18T01:02:03.000000004Z","modified":"1983-04-18T01:02:03.000000004Z","bundle":{"schemaVersion":"schemaVersion","name":"mybun","version":"v0.1.0","description":"this is my bundle","invocationImages":null},"result":{"message":"result message","action":"install","status":"pending"},"parameters":{"myparam":"myparamvalue"},"outputs":{"myoutput":"myoutputvalue"},"custom":["anything goes"]}
```

Comment:
Oh, oops, I think a newline was added, causing a test failure.  (Can also update the test expectation instead, if preferred.)

Reply:
I think I will have to. It appears that my editor keeps inserting the newline on save.

Reply:
Fixed in test so that it is no longer sensitive to newlines at the end of the file